addons/event_hr_skills/models/event_registration.py

```diff
@@ -0,0 +1,42 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models
+
+
+class EventRegistration(models.Model):
+    _inherit = 'event.registration'
+
+    def _create_resume_lines(self):
```

Comment:
Guidelines: fields, compute, CRUD, action methods, tool methods (aka: put that after create / write override)